pkg/validatingadmissionpolicy/builder.go

```diff
@@ -142,6 +142,13 @@ func constructValidatingAdmissionPolicyRules(discoveryClient dclient.IDiscovery,
 		}
 
 		for topLevelApi, apiResource := range gvrss {
+			var resources []string
+			resources = append(resources, apiResource.Name)
+			// if we have pods, we add pods/ephemeralcontainers by default
+			if apiResource.Name == "pods" {
+				resources = append(resources, "pods/ephemeralcontainers")
```

Comment:
Should be print something out to indicate this additional resource is added?

Reply:
I don't think we print a log message when we add in the validatingwebhookconfiguration.